WPMedia/arc-themes-components
# THEMES-1333 calling replace-token-reference from root-tokens

## Ticket

- [THEMES-1333](https://arcpublishing.atlassian.net/browse/THEMES-1333)

## Description
Currently we are handling this issue in the stylebuilder script, but it will be preferred to handle it in the SASS library in arc-themes-components. This will make it easier to maintain our replacement logic in the future.

## Acceptance Criteria

- The replace-token-reference mixin in the arc-themes-components SASS library should also replace uses of $global tokens in the alias definitions.

- The resolveSassTokens function should be removed from the stylebuilder script to keep this functionality in one place.

## Test Steps

1. Checkout branch - `git checkout THEMES-1333-sass-global`
2. link feature-pack repo with fusion/cli
3. start fusion with any block to check the global tokens getting applied.

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1333]: https://arcpublishing.atlassian.net/browse/THEMES-1333?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
